index.php

```diff
@@ -13,22 +13,21 @@
 	<meta name="viewport" content="width=device-width, initial-scale=1.0">
 	<link href="normalize.css" type="text/css" rel="stylesheet" />
 	<link href="styles.css" type="text/css" rel="stylesheet" />
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Prata&display=swap" rel="stylesheet">
 </head>
     <body>
         <header>
                 <h1>CHILLIS</h1>
-                <h2>Welcome to my chilli collection</h2>
         </header>
         <nav>
```

Comment:
at the moment the nav only has an empty div in it. dont think it is doing anything?